src/lib/api/rezdy/fetcher/RezdyFetcher.ts

```diff
@@ -0,0 +1,66 @@
+
+
+
+
+export async function GetRezdySearchResultsFromMarketPlace(searchTerm: string) : Promise<{
+    isSuccessful: true,
+    data: RezdyProductProductSearchResult
+} | {
+    isSuccessful: false,
+    errorMessage: string
+}> {
+    try {
+        const searchItemResponse = await fetch(`${process.env.REZDY_BASE_URL}/v1/products/marketplace?` + new URLSearchParams({
```

Comment:
Could we have a .env.example file with just the keys updated and committed with no values in them. So we know what keys would be needed. 

Possibly have the env values in our notion document?

Reply:
would definitely agree on this..